dbt/adapters/trino/connections.py

```diff
@@ -196,7 +197,8 @@ def open(cls, connection):
             isolation_level=IsolationLevel.AUTOCOMMIT,
             source="dbt-trino",
         )
-        connection.state = "open"
+        trino_conn._http_session.verify = credentials.cert
```

Comment:
I had a look at https://docs.python-requests.org/en/latest/user/advanced/#client-side-certificates and the certificate seems to be set differently. I'm not quite sure that you're setting the right field here.

Reply:
It looks correct based on: 
https://github.com/trinodb/trino-python-client/pull/31